src/semgrep_agent/targets.py

```diff
@@ -376,4 +392,13 @@ def current_paths(self) -> Iterator[List[Path]]:
         :return: A list of paths
         :raises ActionFailure: If git cannot detect a HEAD commit or unmerged files exist
         """
-        yield self._target_paths
+        yield self._paths.targeted
+
+    @property
+    def searched_paths(self) -> List[Path]:
+        """
+        Return list of paths that have been ignored.
```

Comment:
Please change:

The function name, docstring, and actual returned data all disagree with each other